docker/Makefile

```diff
@@ -0,0 +1,18 @@
+GIT_SHOW = $(shell git show --pretty=format:'%H-%ad' --date 'format:%Y%m%dT%H%M%S' | head -n 1)
+GIT_STATUS = $(shell git status --porcelain --ignore-submodules=dirty 2> /dev/null | tail -n1)
```

Comment:
I think you can use `git log -n 1` instead of `git show ... | head -n 1`: 
`git log -n 1 --pretty=format:%ad-%H --date format-local:%Y%m%dT%H%M%S`